websites/B/BladeFM/dist/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+  "author": {
+    "name": "Callum",
+    "id": "506899274748133376"
+  },
+  "service": "BladeFM",
+  "description": {
+    "en": "Providing non-stop music, 24/7",
+    "ga_IE": "Ag soláthar ceoil gan stad, 24/7"
+  },
+  "url": "bladefm.com.au",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Sjj3gx6.png",
+  "thumbnail": "https://i.imgur.com/DWa62sw.png",
+  "color": "#203391",
+  "tags": [
+    "music",
+    "radio",
+    "blade",
+    "bladeradio",
+    "bladefm"
+  ],
+  "category": "music",
+  "settings": [
+    {
+      "id": "timestamp",
+      "title": "Show Timestamp",
+      "icon": "fas fa-history",
+      "value": true
+    },
+    {
+      "id": "details",
+      "title": "Change Line #1 (Song Details)",
+      "icon": "fas fa-music",
+      "value": "Streaming to %listeners% listeners",
+      "placeholder": "%listeners%"
+    },
+    {
+      "id": "state",
+      "title": "Change Line #2 (Presenter Details)",
+      "icon": "fas fa-microphone",
+      "value": "%songText%",
+      "placeholder": "%artist% - %song%"
+    }
```

Comment:
list all available options in placeholder, and allow them to use all in both lines